.github/workflows/python-package.yml

```
@@ -1,41 +1,50 @@
       run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        ruff check .
+        ruff format --check .
+        
+    - name: Type check with pyrefly
+      run: |
+        pyrefly
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)